common/scaffolding/interactive-content-en.html

```diff
@@ -0,0 +1,387 @@
+&lt;/details&gt;</code>
+</pre>
+
+<h2>The <code>dialog</code> element</h2>
+
+<dialog open>Lorem ipsum dolor sit amet, consectetur adipiscing elit.</dialog>
+<pre>
+<code>&lt;dialog open&gt;
+Lorem ipsum dolor sit amet, consectetur adipiscing elit.
+&lt;/dialog&gt;</code>
+</pre>
+
+<h2>The <code>table</code> element</h2>
+
+<table></table>
+<pre>
+<code>&lt;table&gt;&lt;/table&gt;</code>
+</pre>
+
+<h3>2 rows & 2 columns with the <code>tr</code> and <code>td</code> elements</h3>
+
+<table>
+<tr>
+	<td>Lorem ipsum dolor sit amet</td>
+	<td>Proin pellentesque sodales cursus</td>
+</tr>
+<tr>
+	<td>Nunc dictum felis at dignissim dictum</td>
+	<td>Curabitur ut leo nisi</td>
+</tr>
+</table>
+<pre>
+<code>&lt;table&gt;
+&lt;tr&gt;
+	&lt;td&gt;Lorem ipsum dolor sit amet&lt;/td&gt;
+	&lt;td&gt;Proin pellentesque sodales cursus&lt;/td&gt;
+&lt;/tr&gt;
+&lt;tr&gt;
+	&lt;td&gt;Nunc dictum felis at dignissim dictum&lt;/td&gt;
+	&lt;td&gt;Curabitur ut leo nisi&lt;/td&gt;
+&lt;/tr&gt;
+&lt;/table&gt;</code>
+</pre>
+
+<h3>The <code>th</code> element defines a header cell</h3>
+
+<table>
+<tr>
+	<th>Lorem ipsum dolor sit amet</th>
+	<th>Proin pellentesque sodales cursus</th>
+</tr>
+<tr>
+	<td>Nunc dictum felis at dignissim dictum</td>
+	<td>Curabitur ut leo nisi</td>
+</tr>
+</table>
+<pre>
+<code>&lt;table&gt;
+&lt;tr&gt;
+	&lt;th&gt;Lorem ipsum dolor sit amet&lt;/th&gt;
+	&lt;th&gt;Proin pellentesque sodales cursus&lt;/th&gt;
+&lt;/tr&gt;
+&lt;tr&gt;
+	&lt;td&gt;Nunc dictum felis at dignissim dictum&lt;/td&gt;
+	&lt;td&gt;Curabitur ut leo nisi&lt;/td&gt;
+&lt;/tr&gt;
+&lt;/table&gt;</code>
+</pre>
+
+<h3>Group content together with the <code>thead</code>, <code>tbody</code>, <code>tfoot</code> elements</h3>
+
+<table>
+<thead>
+	<tr>
+		<th>Lorem ipsum dolor sit amet</th>
+		<th>Proin pellentesque sodales cursus</th>
+	</tr>
+</thead>
+<tbody>
+	<tr>
+		<td>Nunc dictum felis at dignissim dictum</td>
+		<td>Curabitur ut leo nisi</td>
+	</tr>
+</tbody>
+<tfoot>
+	<tr>
+		<td>Felis dictum nunc at dictum dignissim</td>
+		<td>Leo ut curabitur nisi</td>
+	</tr>
+</tfoot>
+</table>
+<pre>
+<code>&lt;table&gt;
+&lt;thead&gt;
+	&lt;tr&gt;
+		&lt;th&gt;Lorem ipsum dolor sit amet&lt;/th&gt;
+		&lt;th&gt;Proin pellentesque sodales cursus&lt;/th&gt;
+	&lt;/tr&gt;
+&lt;/thead&gt;
+&lt;tbody&gt;
+	&lt;tr&gt;
+		&lt;td&gt;Nunc dictum felis at dignissim dictum&lt;/td&gt;
+		&lt;td&gt;Curabitur ut leo nisi&lt;/td&gt;
+	&lt;/tr&gt;
+&lt;/tbody&gt;
+&lt;tfoot&gt;
+	&lt;tr&gt;
+		&lt;td&gt;Felis dictum nunc at dictum dignissim&lt;/td&gt;
+		&lt;td&gt;Leo ut curabitur nisi&lt;/td&gt;
+	&lt;/tr&gt;
+&lt;/tfoot&gt;
+&lt;/table&gt;</code>
+</pre>
+
+<h3>Format and style entire columns with the <code>colgroup</code> and <code>col</code> elements</h3>
+
+<table>
+<colgroup>
+	<col span="1" style="background-color:red">
+</colgroup>
```

Comment:
We should don't demo example that use inline style.

Instead, you can modify the "Group content together with the <code>thead</code>, <code>tbody</code>, <code>tfoot</code> elements" to include also the grouping element "colgroup" and "col". You can rename that title like "Grouping tabular cells"

Reply:
Fixed in commit "Addressed changes".